pcp/Platform.c

```diff
@@ -699,6 +703,17 @@ bool Platform_getNetworkIO(NetworkIOData* data) {
    return true;
 }
 
+void Platform_getFileDescriptors(double* used, double* max) {
+   *used = NAN;
+   *max = 65536;
```

Comment:
I am not yet familiar with PCP platform, but would you please comment on where this magic number is from?